transaction-status/src/lib.rs

```diff
@@ -128,6 +143,16 @@ impl UiInnerInstructions {
     }
 }
 
+impl UiTransactionTokenBalance {
+    fn parse(token_balance: TransactionTokenBalance) -> Self {
+        Self {
+            account_index: token_balance.account_index,
+            mint: token_balance.mint,
+            ui_token_amount: token_balance.ui_token_amount,
+        }
+    }
+}
+
```

Comment:
I don't think we need this special parse function, do we? I think the From implementations between UiTransactionTokenBalance and TransactionTokenBalance that I mention elsewhere should suffice?